document-en.md

```diff
@@ -0,0 +1,244 @@
+## FramgiaCI Documents
+`Sun*CI` is a CI/CD system developed by Sun* Asterisk , beside the main features like the other CI/CD system, `Sun*CI` provided more features for assisting user in developing projects.Here's some  instruction you better look before intergrate with `Sun*CI`
```

Comment:
`Here's some  instruction` => `Here's some  instructions` 

`you better` => `you'd better`